slurm-on-gke/.gitignore

```diff
@@ -0,0 +1,6 @@
+
+.terraform*
+terraform.tfstate
+terraform.tfstate.backup
+terraform.tfvars
+*/.terraform
```

Comment:
Can you add new lines at the end of each file? I noticed this warning in other files too

Reply:
of course, I'll push this in a bit